src/libs/antares/study/parts/short-term-storage/container.cpp

```diff
@@ -142,21 +142,27 @@ static bool loadAdditionalConstraintsProperties(AdditionalConstraints* additiona
                 return false;
             }
         }
+        else
+        {
+            logs.error() << "Constraint " << additionalConstraints->name << " : "
+                         << "has a wrong key : " << key;
+            return false;
+        }
     }
     return true;
 }
 
 bool STStorageInput::loadAdditionalConstraints(const fs::path& parentPath)
```

Comment:
This function could be easily split into 2 parts : 
- load additional constraints properties from ini file
- load additional constraints associated RHS